### PR TITLE
JWST B7.5 compatibility updates 

### PR DESCRIPTION
**Pull request by vcgeers**
This PR introduces changes needed to make MiriTE compatible with JWST PL B7.5.

The changes include:
- Reduce RSCD test to minimum to make the MIRI data models more resilient to changes.

### CODE REVIEW

File: miri/datamodels/tests/test_reset_switch_charge_decay_model.py [hunk 1]
```diff
@@ -122,11 +85,11 @@ def test_fitsio(self):
             self.dataproduct.save(self.testfile, overwrite=True)
             with MiriResetSwitchChargeDecayModel(self.testfile) as readback:
                 self.assertIsNotNone(readback.rscd_table)
```
Comment (vcgeers) [inline miri/datamodels/tests/test_reset_switch_charge_decay_model.py:87]: I have tried out this branch with my local B7.5 based MIRICLE build, and for this line 87, it throws the following error
E               AttributeError: No attribute 'rscd_table'

Looking at the latest JWST rcsd schema file at:
https://github.com/spacetelescope/jwst/blob/master/jwst/datamodels/schemas/rscd.schema.yaml
it appears that they have now defined properties such as rscd_group_skip_table, rscd_gen_table, rscd_int1_table, etc.

Should we update the test to assert for one of those instead?

File: miri/datamodels/tests/test_reset_switch_charge_decay_model.py [hunk 2]
```diff
@@ -83,33 +61,18 @@ def test_referencefile(self):
...
-        self.assertIsNotNone(nulldp.rscd_table)
-        descr2 = str(nulldp)
-        self.assertIsNotNone(descr2)
-        del nulldp, descr1, descr2    
+        del nulldp, descr1
 
     def test_copy(self):
         # Test that a copy can be made of the data product.
         datacopy = self.dataproduct.copy()
         self.assertIsNotNone(datacopy.rscd_table)
```
Comment (vcgeers) [inline miri/datamodels/tests/test_reset_switch_charge_decay_model.py:74]: See comment for line 87 for context: this line 74 is the 2nd place where the same error occurs.

**Comment by vcgeers** [comment]
Steven, as an experiment, I update the above-mentioned 2 lines to replace ".rscd_table" with ".rscd_group_skip_table", and that appears to resolve the issue; after that change, all 5 tests in that test module pass successfully.

I hope you can comment on whether you want to indeed proceed that way, or have a different solution in mind.

**Comment by smbeard** [comment]
The problem should now be solved. I accidentally left a dependency in the unit test. The best thing is just to remove the dependency, otherwise we will spend our time modifying this unit test to keep up with changes outside our control.

**Comment by vcgeers** [comment]
Thank you for the rapid changes Steven. Agreed, best to leave those out to avoid chasing those model changes.

Steven confirmed that he tested this in his B7.4 MIRICLE, and I just tested these in my local B7.5 based MIRICLE and confirmed that the unit tests now pass ok.

Will proceed with merging this into master.

**Pull request merged**